docs/document/content/reference/distsql/syntax/rdl/resource-definition/alter-resource.en.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "ALTER RESOURCE"
+weight = 3
++++
+
+
+### Description
+
+The `ALTER RESOURCE` syntax is used to alter resources for the currently selected schema.
+
+
+
+### Syntax
+```SQL
+AlterResource ::=
+  'ALTER' 'RESOURCE' dataSource (',' dataSource)*
+
+dataSource ::=
+  dataSourceName '(' ( 'HOST' '=' hostName ',' 'PORT' '=' port ',' 'DB' '=' dbName  |  'URL' '=' url  ) ',' 'USER' '=' user (',' 'PASSWORD' '=' password )?  (',' 'PROPERTIES'  '(' ( key  '=' value ) ( ',' key  '=' value )* ')'  )?')'
+
+dataSourceName ::=
+  identifier
+
+hostname ::=
+  identifier | ip
+
+dbName ::=
+  identifier
+
+port ::=
+  int
+
+password ::=
+  identifier | int | string 
+
+user ::=
+  identifier
+
+url ::=
+  identifier | string
+
+```
+
```

Comment:
Could we add an `Example` section here ? Just like `ADD RESOURCE`
 https://shardingsphere.apache.org/document/current/en/reference/distsql/syntax/rdl/resource-definition/add-resource/

Reply:
Ofcourse, do you want to provide an example?

Reply:
Referring the syntax in the `g4` file for `ALTER RESOURCE` => https://github.com/apache/shardingsphere/blob/master/shardingsphere-distsql/shardingsphere-distsql-parser/src/main/antlr4/imports/RDLStatement.g4#L26

You can use the same examples from `ADD RESOURCE` Just change `ADD RESOURCE` to `ALTER RESOURCE` in the examples .